tests/QAN/overview_test.js

```diff
@@ -9,6 +11,27 @@ Before(async ({
   qanFilters.waitForFiltersToLoad();
 });
 
+Scenario(
+  'PMM-T207 Verify hovering over query in overview table  @qan',
+  async ({ I, qanOverview }) => {
+    qanOverview.waitForOverviewLoaded();
+    const firstQueryLocator = `${qanOverview.getRowLocator(1)} div.td:nth-child(2) div`;
```

Comment:
do you have any specific reason for including all these inside the scenario file? why can't locators and logic go to overview_fragment, thats how it works for all other scenarios too.